clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -7588,8 +7588,6 @@ static void handleSYCLIntelMaxReplicatesAttr(Sema &S, Decl *D,
 /// second is a direction.  The direction must be "depth" or "width".
 /// This is incompatible with the register attribute.
 static void handleSYCLIntelMergeAttr(Sema &S, Decl *D, const ParsedAttr &AL) {
-  checkForDuplicateAttribute<SYCLIntelMergeAttr>(S, D, AL);
```

Comment:
Can we get rid of this? Or is it being called elsewhere? Is this community code?

Reply:
This is not a community code. We added this part of FPGA attribute support. IT has been used by other FPGA attributes. 
```
checkForDuplicateAttribute<SYCLIntelSinglePumpAttr>
checkForDuplicateAttribute<SYCLIntelDoublePumpAttr>
checkForDuplicateAttribute<SYCLIntelMemoryAttr>
checkForDuplicateAttribute<SYCLIntelRegisterAttr>
checkForDuplicateAttribute<SYCLIntelSimpleDualPortAttr>
checkForDuplicateAttribute<SYCLIntelBankBitsAttr>

```
I will remove it once we fix duplicate attributes bugs with them.